matador/plotting/ir_plotting.py

```diff
@@ -0,0 +1,126 @@
+""" This submodule implements functions useful for plotting
+the results of infrared and Raman spectroscopy calculations.
+
+"""
+
+from matador.plotting.plotting import plotting_function
+
+
+def read_ir_file(seed):
+    """ This function should take the seed filename and
+    read in the IR data, probably into a dictionary.
+
+    Parameters:
+        seed (str): the filename to read.
+
+    Returns:
+        dict: containing IR data
+
+    """
+
+    with open(seed + '.phonon', 'r') as f:
+        flines = f.readlines()
+    for ind, line in enumerate(flines):
+        if 'Number of branches' in line:
+            no_branches = int(line.split()[3])
+        if 'END header' in line:
+            header_end = ind
+
+    # contains IR and Raman data
+    ir_dat = dict()
+    for line in flines[header_end+2:header_end+no_branches+2]:
+        line_split = line.split()
+        if len(line_split) < 2:
+            print_failure('Failed to read IR data. Check .phonon file.')
+            exit()
```

Comment:
I'm guilty of this throughout the code, but it's probably better to raise a `RuntimeError` here rather than hard exiting, to make the function more reusable.

```suggestion
            raise RuntimeError('Failed to read IR data. Check .phonon file.')
```